tests/data/dataconverter/readers/apm/nomad_oasis_eln_schema_for_nx_apm/nxapm.schema.archive.yaml

```diff
@@ -3,16 +3,12 @@
 # quantity in NXapm
 definitions:
   name: 'apm'
-  # 'ELN/application definition schema for atom probe microscopy (APM) experiments.'
-  sections: # section definitions what were back in the old days msection base classes
-    # Operator:
-    # Specimen:
+  # 'ELN/matching NXapm NeXus appdef data schema for atom probe'
+  sections: # section definitions what were back in the old days msections
     AtomProbeMicroscopy: # the actual schema
-      # nomad.datamodel.metainfo.eln.NexusParser
       base_sections:
-        - 'nomad.datamodel.metainfo.eln.NexusParser'
+        - 'nomad.datamodel.metainfo.eln.NexusDataConverter'
```

Comment:
shall it be modified in the other ELN schemas when updating the NOMAD modules?

Reply:
Yes, this function was renamed as I can confirm also using the update-north nomad-FAIR branch on the 2023/05/28 (with which I am testing the examples)